src/component/admin/apiToken/ApiTokenForm/ApiTokenForm.tsx

```diff
@@ -40,13 +49,31 @@ const ApiTokenForm: React.FC<IApiTokenFormProps> = ({
     clearErrors,
 }) => {
     const TYPE_ADMIN = 'ADMIN';
+    const { uiConfig } = useUiConfig();
     const { classes: styles } = useStyles();
     const { environments } = useEnvironments();
     const { projects: availableProjects } = useProjects();
 
     const selectableTypes = [
-        { key: 'CLIENT', label: 'Client', title: 'Client SDK token' },
-        { key: 'ADMIN', label: 'Admin', title: 'Admin API token' },
+        {
+            key: 'CLIENT',
```

Comment:
Could perhaps use an array of strings instead, since keys and labels are the same:

```ts
const selectableTypes = uiConfig.embedProxy
    ? ['CLIENT', 'ADMIN', 'FRONTEND']
    : ['CLIENT', 'ADMIN'];
```

...and treat `title` like `getTypeDescription`. Though I suppose the labels might change.

Reply:
It's no longer consistent between table and form.